spec/requests/api/v2/build_new_course_spec.rb

```diff
@@ -42,7 +42,6 @@
   context "with subjects" do
     let(:params) do
       { course: {
-        name: "Foo Bar Course",
```

Comment:
Under normal circumstance the client is not allowed to send `name` unless they are an admin. The previous code use to accept it then ignore it.